docs/en/week12/12-2.md

```diff
@@ -0,0 +1,335 @@
+---
+lang-ref: ch.13-2
```

Comment:
Is this week 13 or 12???

Reply:
I'm confused I thought I was responsible for 12-2 only. The summary is in the scribe notes. Apologies for the wrong lang-ref.